deploy/03-rbac.yaml

```diff
@@ -0,0 +1,72 @@
+kind: Role
```

Comment:
for consistency should probably put --- up here too, though i guess it's implicit.

Reply:
yeah it is / wasn't there before